src/Core/Form/IdentifiableObject/DataFormatter/BulkCombinationFormDataFormatter.php

```diff
@@ -84,6 +84,19 @@ public function format(array $formData): array
             }
         }
 
+        // We only update images if disabling_switch_images value is truthy
```

Comment:
Is it a typo ? I think you meant `if disabling_switch_images value is true`

Reply:
no, truthy is what i meant. It means any true-like value, we don't care much if its strictly boolean "true" or 1, or "1", which is a possibilty because it is a form data. (though it probably will always be true/false if disablingSwitch component works as expected).